app-forensics/aide/files/aide-0.16-respect-configure-options.patch

```diff
@@ -0,0 +1,103 @@
+commit 8f16b2518a4f77d9da453a2d5dba78aeb7818ad3
+Author: Ilya Tumaykin <itumaykin@gmail.com>
+Date:   Tue May 23 13:28:46 2017 +0300
+
+configure: respect user choices for every option
+
+Otherwise configuration becomes automagic fiesta, which is very bad.
+See https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Automagic_dependencies
+and https://sourceforge.net/p/aide/bugs/97/
+
+diff --git a/configure.ac b/configure.ac
+index 3598ebe..5aa36cc 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -263,7 +263,7 @@ AC_CHECK_FUNCS(stricmp strnstr strnlen)
+ AC_ARG_WITH([mmap],
+ 	[AC_HELP_STRING([--with-mmap],
+ 		[use mmap @<:@default=check@:>@])],
+-	[],
++	[with_mmap="$withval"],
```

Comment:
This is needed for posix-acl configure option that maps to acl USE.
Then I went ahead and updated other options with this pattern to match the rest.